### DISCUSSION TITLE
IKE_INTERMEDIATE

### DISCUSSION THREAD

**Discussion by keskinadem** [strongswan/strongswan#2115] [Q&A] [answered]
When I read the RFC 9242 Acknowledgements part, I understand that IKE_INTERMEDIATE idea is used for general purpose between IKE_SA_INIT and IKE_AUTH exchanges. I used strongswan for multiple Key exchange and see IKE_INTERMEDIATE exchanges and can ı use IKE_INTERMEDIATE exchange to negotiate something else(except additional key exchange)? Does strongswan implementation allows this?

**Comment by tobiasbrunner** [comment, marked as answer]
No, it currently does not. What kind of other use do you have in mind?
